android/app/src/main/java/com/reactnativenavigation/activities/BottomTabActivity.java

```diff
@@ -64,10 +64,9 @@ private void setupPages(ArrayList<Screen> screens) {
 
     private void setupToolbar(ArrayList<Screen> screens) {
         Screen initialScreen = screens.get(0);
-        setNavigationStyle(initialScreen);
         mToolbar.setScreens(screens);
-        mToolbar.setTitle(initialScreen.title);
-        setSupportActionBar(mToolbar);
+        mToolbar.updateToolbar(initialScreen);
+        setNavigationStyle(initialScreen);
```

Comment:
Sure, we'll address that in a separate PR. I'll merge soon. Thanks for the contribution Matt 👍